mungegithub/github/github.go

```diff
@@ -1557,21 +1557,20 @@ func (obj *MungeObject) ListFiles() ([]*github.CommitFile, bool) {
 }
 
 // GetPR will return the PR of the object.
-func (obj *MungeObject) GetPR() (*github.PullRequest, bool) {
+func (obj *MungeObject) GetPR() *github.PullRequest {
 	if obj.pr != nil {
-		return obj.pr, true
+		return obj.pr
 	}
 	if !obj.IsPR() {
-		glog.Errorf("Issue: %d is not a PR", *obj.Issue.Number)
-		return nil, false
+		return nil
 	}
 	pr, err := obj.config.getPR(*obj.Issue.Number)
 	if err != nil {
-		glog.Errorf("Error in GetPR")
-		return nil, false
+		glog.Errorf("Error in GetPR: %v", err)
+		return nil
```

Comment:
Above, under `if !obj.IsPR()` it also returns the same 'return nil, false' problem. So you can't distinguish between 'isn't a PR' and 'github API error' from the return value of this function in either case.